src/goto-instrument/goto_instrument_parse_options.h

```diff
@@ -51,7 +51,7 @@ Author: Daniel Kroening, kroening@kroening.com
   "(show-struct-alignment)(interval-analysis)(show-intervals)" \
   "(show-uninitialized)(show-locations)" \
   "(full-slice)(reachability-slice)" \
-  "(inline)(remove-function-pointers)" \
+  "(inline)(partial-inline)(function-inline):(log):" \
```

Comment:
The rebase seems to have introduced a bug here: the "remove-function-pointers" option should not go away!